module/dkg/broker.go

```diff
@@ -15,6 +16,13 @@ import (
 	"github.com/onflow/flow-go/utils/hasher"
 )
 
+// RETRY_MAX is the maximum number of times the broker will attempt to broadcast
+// a message or publish a result
+const RETRY_MAX = 20
```

Comment:
```suggestion
const RETRY_MAX = 5
```

In a real network it may take quite a while before it is clear that a transaction submission has failed (eg. 10 mins for an expiry failure) so I'd rather more aggressively limit the retries.